src/adapter_test_helpers/sumo_events_test_helper.erl

```diff
@@ -11,13 +11,13 @@
 
 -spec events_manager_supervisor_running(Config :: config()) -> ok.
 events_manager_supervisor_running(Config) ->
-  {_, Module} = lists:keyfind(module, 1, Config),
+  {_, Name} = lists:keyfind(name, 1, Config),
```

Comment:
I wouldn't make this change, at least for now. Because the `events` config hasn't changed, it is still the same (`{doc_module, event_manager_module}`):

```
{events, [
  {sumo_test_people_mnesia, sumo_test_people_mnesia_events_manager}
]}
```

I'd propose you create an issue related to this, e.g.: "Modify events config to follow the same form as docs – `{model, event_manager_module}` instead of `{doc_module, event_manager_module}`".